package.json

```diff
@@ -29,24 +29,23 @@
     "lint": "lerna run lint --stream",
     "lint:changes": "yarn lint --since HEAD^",
     "lint:fix": "lerna run lint:fix --stream",
-    "lint:commit": "ui-scripts --lint-commit",
+    "lint:commit": "ui-scripts lint-commit",
```

Comment:
i had to rewrite some of our internal commands. `yargs` does not accept command names starting with `--`